site/en/docs/extensions/mv3/manifest/manifest_version/index.md

```diff
@@ -2,27 +2,23 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest Version"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2021-04-19
 description: Reference documentation for the manifest_version property of manifest.json.
 ---
 
-One integer specifying the version of the manifest file format your package requires. As of Chrome
-18, developers _should_ specify `2` (without quotes) to use the format as described by this
-document:
+An integer specifying the version of the manifest file format your package requires. For example:
 
 ```json
-"manifest_version": 2
+"manifest_version": 3
 ```
 
-Consider manifest version 1 _deprecated_ as of Chrome 18. Version 2 is not yet _required_, but we
-will, at some point in the not-too-distant future, stop supporting packages using deprecated
-manifest versions. Extensions, applications, and themes that aren't ready to make the jump to the
-new manifest version in Chrome 18 can either explicitly specify version `1`, or leave the key off
-entirely.
+Legal values for this key are:
 
-The changes between version 1 and version 2 of the manifest file format are described in detail in
-[the `manifest_version` documentation.][1]
+* 3: Use the [Manifest V3][mv3] format and associated feature set.
+* 2: Use the [Manifest V2][mv2] format and associated feature set.
 
-<div class="aside aside--caution">Setting <code>manifest_version</code> 2 in Chrome 17 or lower is not recommended. If your extension needs to work in older versions of Chrome, stick with version 1 for the moment. We'll give you ample warning before version 1 stops working.</div>
+The current version is Manifest V3. Manifest V2 is also currently permitted, but will be phased out
+in the near future. Other manifest versions greater than 3 are not planned yet.
```

Comment:
We have been rightly called out elsewhere for this vague timing. I think we should be better at clarifying our position as to when MV2 will be deprecated, or rather, how long developers will have to migrate once we announce MV2's deprecation (e.g. I vaguely remember but cannot find a blog post saying we'll give developers a year).

Reply:
Yep, we're working on generating a concrete timeline for developers.  Once that happens, we should include it here.  I don't think we need to block on that for this update, though

Reply:
Isn't it too early to promote MV3 as the "current" version? It's still in development and full of bugs. Currently even the most basic things are partially and unpredictably broken e.g. chrome.runtime.sendMessage or chrome.storage.local.get methods don't return a Promise. Judging by the amount of time it took to come this far, MV3 will need at least a year to become "current".